rclcpp/minimal_action_server/not_composable.cpp

```diff
@@ -49,18 +49,18 @@ void execute(
   RCLCPP_INFO(rclcpp::get_logger("server"), "Executing goal");
   rclcpp::Rate loop_rate(1);
   const auto goal = goal_handle->get_goal();
-  auto feedback = std::make_shared<Fibonacci::Feedback>();
-  auto& sequence = feedback->sequence;
+  auto feedback = std::make_shared<Fibonacci::Impl::FeedbackMessage>();
```

Comment:
@dirk-thomas this seems off, the user shouldn't have to fiddle with implementation details.

Reply:
Resolved in 3c2b2e804c1395cdb289365291b30b2050ddb610 after changing the API signature in `rclcpp_action`.